### ISSUE TITLE
Is possible to compress in admin models?

### ISSUE THREAD

**Issue by mamcx** [django-compressor/django-compressor#883]
I have a Admin model with extra JS:

    class Media:
        js = (
            '/static/js/jquery.maskedinput.min.js',
            '/static/js/moment-with-locales.min.js',
            '/static/js/knockout-3.2.0.js',
            '/static/js/knockout.validation.min.js',
            '/static/js/decimal.min.js',
            '/static/js/accounting.js',
            '/static/js/main.js',
            '/static/js/gastos.js',

Is possible to use compresor for this?

**Comment by karyon** [comment]
i don't think so. most likely, none of us maintainers will implement this, but you are welcome to submit a pull request.

**Comment by mamcx** [comment]
Is possible to run manually? like run compresor with a list of links in the command-line?

**Comment by karyon** [comment]
well, even if we had that, django would still need to know where the results are when rendering the admin pages... a.k.a. i'm not aware of a workaround, sorry :(
